packages/backend/src/core/activity/postprocessCounts.ts

```diff
@@ -15,7 +15,7 @@ export function postprocessCounts(
   counts: DailyTransactionCount[],
   processedAll: boolean,
   now?: UnixTime,
-) {
+): DailyTransactionCount[] {
   const today = (now ?? UnixTime.now()).toStartOf('day')
```

Comment:
Why this doesn't use the clock?

Reply:
Maybe I am not getting something, but I don't see a reason why it should use clock - `DailyTransactionCount.timestmap` is an actual timestamp, not clock-relative and in this function we are interested in relation between timestamp and actual time.

Reply:
Yes and I thought we are using clock to get the actual time and it returns the offset, so that the entire system can work with that.